Gododir/gen.go

```diff
@@ -50,14 +50,22 @@ var (
 					Version: "2016-05-16",
 				},
 				{
-					Name:    "authorization",
-					Version: "2015-07-01",
-				},
-				{
-					Name:    "batch",
-					Version: "2015-12-01",
-					Swagger: "BatchManagement",
+					Name:    "apimanagement",
```

Comment:
@amarzavery Do you know if API management will have a composite swagger?
@marstr If this service wont have a composite swagger, I would follow the same approach as we did with machinelearning and resources.